pkg/pb/contact.proto

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+option go_package = "pb";
+
+package contact;
+
+import "gogo.proto";
+
+service ContactNode {
```

Comment:
The package is already called `contact` so the full name would end up being `contact.ContactNode`.